lib/rubygems/commands/owner_command.rb

```diff
@@ -98,8 +98,10 @@ def manage_owners(method, name, owners)
         action = method == :delete ? "Removing" : "Adding"
 
         with_response response, "#{action} #{owner}"
-      rescue
-        # ignore
+      rescue Gem::WebauthnVerificationError => e
+        raise e
+      rescue StandardError
+        # ignore early exits to allow for completing the iteration of all owners
```

Comment:
Got some conflicts here due to missing RuboCop changes. Hopefully I resolved them correctly!